app/views/layouts/base.html.erb

```diff
@@ -54,7 +54,7 @@
       <% end %>
     <% end %>
 
-    <a href="#main-content" class="govuk-skip-link">Skip to main content</a>
```

Comment:
Just wondering, should this be the first focussed thing when tabbing? At the moment, if the cookie banner is shown, then its buttons are focussed first

Reply:
I would have thought so too, but [the guidance says](https://design-system.service.gov.uk/components/cookie-banner/#how-it-works):

> Position the cookie banner after the opening <body> tag and before the ‘skip to main content’ link.